### PR TITLE
Add clearer usage instructions and do some minor formatting

### PR DESCRIPTION
**Pull request by ayyrickay**
Based off of @friederbluemle's instructions in https://github.com/pxb1988/dex2jar/issues/170

### CODE REVIEW

**Review by friederbluemle — APPROVED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# dex2jar 
+
 **Project move to [SourceForge](https://sourceforge.net/p/dex2jar) and [Bitbucket](https://bitbucket.org/pxb1988/dex2jar) and [Github](https://github.com/pxb1988/dex2jar)**
```
Comment (friederbluemle) [inline README.md]: Can you also fix the typo here please: `GitHub`

Comment (ayyrickay) [reply README.md]: Good catch. (: Fixed!

**Review by SkarDude — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
-**Project move to [SourceForge](https://sourceforge.net/p/dex2jar) and [Bitbucket](https://bitbucket.org/pxb1988/dex2jar) and [Github](https://github.com/pxb1988/dex2jar)**
+# dex2jar
```
Comment (friederbluemle) [inline README.md]: Please remove the trailing space character on this line.

Comment (ayyrickay) [reply README.md]: Trailing space removed 👍

**Comment by mfalkvidd** [comment]
Thanks! Looked all over the repo trying to figure out how to get started. These instructions are very helpful.

**Comment by pxb1988** [comment]
i have merged a similar patch, close the request, Thanks !

#128

**Pull request merged**